sequence_processing_pipeline/QCJob.py

```diff
@@ -411,18 +405,41 @@ def _gen_chained_cmd(self, current_dir, filename1, filename2,
         path2 = join(partial, filename2.replace('.fastq.gz',
                                                 '.trimmed.fastq.gz'))
 
+        kraken_report_path = join(partial,
+                                  filename1.replace('.fastq.gz',
+                                                    '.kraken2_report.txt'))
+
+        kraken_output_path = join(partial,
+                                  filename1.replace('.fastq.gz',
+                                                    '.kraken2.trimmed.#.fastq')
+                                  )
+
         result = self.fastp_path
+
         if adapter_a:
             # assume that if adapter_a is not None, then adapter_A is not None
             # as well. We performed this check already.
             result += (f' --adapter_sequence {adapter_a}'
                        f' --adapter_sequence_r2 {adapter_A}')
 
-        result += (f' -l 100 -i {read1_input_path} -I {read2_input_path} '
-                   f'-w {self.nprocs} -j {json_output_path} -h '
-                   f'{html_output_path} --stdout | {self.minimap2_path} -ax'
-                   f' sr -t {self.nprocs} {self.mmi_db_path} - -a | '
-                   f'{self.samtools_path} fastq -@ {self.nprocs} -f 12 -F '
-                   f'256 -1 {path1} -2 {path2}')
+        result += (f' -l 100 -i {read1_input_path} -I {read2_input_path} -w '
```

Comment:
@charles-cowar, could you open an issue to define the min length of the reads via the config file or somewhere else that is not hardcoded? Thank you.

Reply:
Done! https://github.com/biocore/mg-scripts/issues/76